frontend/src/components/headerSection/header.css

```diff
@@ -8,7 +9,7 @@
     background-image: url(../../assets/backgrounds/header_bg.png);
     background-color: #fffde2;
     background-repeat: no-repeat;
-    background-size: cover;
+    background-size: 100% auto;
```

Comment:
Reason I want cover is because of this:
![image](https://user-images.githubusercontent.com/6146988/112248989-f5792280-8c24-11eb-960f-7c64c7a1ee0e.png)